src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -107,12 +107,14 @@
 	private Boolean enabled;
 	private Boolean seeded;
 
-	private final boolean use_cdm;
 	private final String cdm_project_prefix;
 
+    private KibanaSeed kibanaSeed;
```

Comment:
Do you need this to be a member of the class?  You only reference it in one place, in the same context where it is used.